### DISCUSSION TITLE
Welcome to shark Discussions!

### DISCUSSION THREAD

**Discussion by lau1944** [lau1944/shark#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by iamngoni** [comment]
Hi 👋. I'm interested in contributing to this project. Mind sharing more info on your full intentions

**Reply by lau1944** [reply]
It's a server rendering framework by parsing json format data. After did some code reading on [dynamic_widget](https://pub.dev/packages/dynamic_widget) source code, I decided to use it instead.

So the work here now mainly focus on downloading the json and make some optimization. I also plan to create a npm module called shark_parser, which simplify the process of create json string. 

After finishing the work, i would test it, if the performance is acceptable, I would release the package, or I will figure out another way to parse data.
